docs/ref/models/querysets.txt

```diff
@@ -1269,6 +1269,18 @@ where prefetching with a custom ``QuerySet`` is useful:
     specific order to avoid creating extra queries; therefore it's recommended
     to always carefully order ``prefetch_related`` arguments.
 
+.. note::
```

Comment:
I'm not sure if docs are needed here. Does `Model.refresh_from_db()` need an update or is it fine?